DOCS-3390: Apply Micro-RDK feedback (#3896)
viamrobotics/docs: docs/operate/get-started/supported-hardware/_index.md

### For use with ESP-32 microcontrollers

The following is a selection of components (some built-ins and some modules) written for use with the Micro-RDK.
To use any of the built-in components, configure them according to their readmes.

> To use a module with the Micro-RDK, you can either install the pre-built `viam-micro-server` firmware that ships with a few common modules, or [build your own firmware that combines the Micro-RDK with one or more modules](/operate/get-started/other-hardware/micro-module/).

Reviewer: ```suggestion
You can either install the pre-built `viam-micro-server` firmware that ships with a few common modules, or [build your own firmware that combines the Micro-RDK with one or more modules](/operate/get-started/other-hardware/micro-module/).
```

Author: ```suggestion
You can either install the pre-built `viam-micro-server` firmware that ships with a few common modules, or [build your own firmware that combines the Micro-RDK with one or more modules of your choice](/operate/get-started/other-hardware/micro-module/).
```
Okay! Also realizing maybe this makes more sense

Reviewer: LGTM

In addition to physical hardware, there are "virtual" hardware modules that do not directly drive any physical hardware, but rather augment physical hardware with another layer of abstraction, or add other functionality, for example:

> - [A "camera" that takes a camera feed from a physical camera, and crops it, overlays it, or otherwise transforms the output](/operate/reference/components/camera/transform/)

Reviewer: Curious if this is being intentionally removed?

Author: This was Rand feedback--that this particular module is wonky (hogs CPU) so maybe not the best example to showcase. I realize I'm replacing it with a built-in that's not actually a module, but hopefully still a good example of an abstracted component.

Reviewer: Gotcha, sounds good

- [A "movement sensor" that calculates the estimated the position of a wheeled rover based on the output of other components](/operate/reference/components/movement-sensor/wheeled-odometry/)
- [A ChatGPT integration module](https://github.com/jeremyrhyde/chat-gpt-module)

> These modules implement the same [component APIs](/dev/reference/apis/#component-apis) as physical hardware modules, and are configured in the same way as other components.

Reviewer: Nice, this feels a lot more smooth
